awx/main/analytics/collectors.py

```diff
@@ -52,7 +52,6 @@ def config(since, **kwargs):
         'instance_uuid': settings.SYSTEM_UUID,
         'tower_url_base': settings.TOWER_URL_BASE,
         'tower_version': get_awx_version(),
-        'ansible_version': get_ansible_version(),
```

Comment:
cc @Ladas

Reply:
@beeankha even though this isn't reported here anymore, it's now part of the analytics payload we ship for jobs (per-job) 🎉